src/test/java/seedu/address/model/tuition/TuitionTest.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.model.tuition;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
+import static seedu.address.testutil.TypicalStudents.ALICE;
+import static seedu.address.testutil.TypicalStudents.BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.session.Session;
+import seedu.address.model.student.Student;
+import seedu.address.testutil.SessionBuilder;
+import seedu.address.testutil.StudentBuilder;
+
+class TuitionTest {
```

Comment:
How about a test case to consider when all/some parameters are null? 
`new Tuition(null, null, null, null)`

Reply:
Ah good point will make changes!